app/models/ability.rb

```diff
@@ -18,7 +18,7 @@ def custom_permissions
       end
       cannot :create, ::Collection 
       cannot :destroy, ::Collection unless admin?
-      cannot :manage, ::Collection do |obj|
+      cannot [:edit, :update], ::Collection do |obj|
```

Comment:
manage is ANY action.  Do we have to worry about create, destroy or any other actions here?